.github/workflows/test.yml

```
@@ -85,3 +85,49 @@ jobs:
 
           echo "Use this command to try out the build:"
           echo "  conda install -c ${ANACONDA_ORG_CHANNEL}/label/${ANACONDA_ORG_LABEL} conda-launchers"
+
+  rattler_build_recipe:
+    name: Rattler build recipe
+    runs-on: windows-latest
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      - name: Setup Pixi
+        uses: prefix-dev/setup-pixi@v0.8.1
+        with:
+          cache: true
+
+      - name: Run Build
+        shell: pwsh
+        run: |
+          pixi run build build --recipe-dir conda-launchers `
+            --log-style plain `
+            --skip-existing all `
+            -c conda-forge -c conda-canary/label/dev
+
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: conda-launchers-builds
+          path: |
+            ./output/noarch/conda-launchers*.conda
+
+      - name: Upload to Anaconda
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        env:
+          ANACONDA_OWNER: conda-canary
+          ANACONDA_CHANNEL: dev
```

Can we use a different label here so the two versions don't clash? 
```suggestion
          ANACONDA_CHANNEL: dev-zig
```